lib/xlint.rb

```diff
@@ -28,6 +29,8 @@ def check_env
 
     def build_draft
       @comments = []
+      # GitDiffParser::Patches.parse(cp932 text) raises ArgumentError: invalid byte sequence in UTF-8
```

Comment:
[canvas-lms](https://github.com/instructure/canvas-lms/blob/58912ddfb899a1b1c151c3c64e872b13300468c8/gems/rubocop-canvas/lib/rubocop_canvas/helpers/diff_parser.rb) appears to have their own parser.